resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -64,3 +64,18 @@ function createListElement(date, type, text, id) {
 
   return liElement;
 }
+
+/**
+ * 
+ */
+async function getRevieweeResume() {
+  fetch('/blobstore-serve').
+    then(response => response.arrayBuffer())
+    .then((pdfArrayBuffer) => {
+            var adobeDCView = new AdobeDC.View({clientId: "b98bbf69d44442479396583253ac267c", divId: "adobe-dc-view"});
+            adobeDCView.previewFile({
+              content:{promise: pdfArrayBuffer },
+              metaData:{fileName: "reviewee.pdf"}
+            }, {});
```

Comment:
From the error message in your screenshot, it looks like `previewFile` wants a Promise that resolves in an ArrayBuffer. Your current code is passing it an ArrayBuffer, instead of a Promise for an ArrayBuffer. Try this:

```
fetch('/blobstore-serve')
  .then((response) => {
            var adobeDCView = new AdobeDC.View({clientId: "b98bbf69d44442479396583253ac267c", divId: "adobe-dc-view"});
            adobeDCView.previewFile({
              content: {promise: response.ArrayBuffer()},
              metaData: {fileName: "reviewee.pdf"}
            }, {});
```

Reply:
I'm getting an TypeError now saying `response.ArrayBufffer()` is not a function
![Screenshot 2020-07-15 at 9 54 27 AM](https://user-images.githubusercontent.com/44929204/87560562-37f36400-c681-11ea-9e0d-141b1d2a1c0b.png)

Reply:
Sorry, it should be `response.arrayBuffer()` instead of `response.ArrayBuffer()`.

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Body/arrayBuffer